docs/source/tutorial/package_a_project.md

```diff
@@ -4,25 +4,96 @@ This section explains how to build your project documentation, and how to bundle
 
 Kedro also has an advanced feature which supports packaging on a pipeline level allowing you share and reuse pipelines across projects! To read more about this please look at the [section on micro-packaging](../nodes_and_pipelines/micro_packaging.md).
 
-
 ## Add documentation to your project
 
-Kedro uses the [Sphinx framework](https://www.sphinx-doc.org) and creates a `docs` directory that builds a basic template for project-specific documentation. We recommend that you add your project-specific documentation as markdown in `docs/source`
+Kedro uses the [Sphinx framework](https://www.sphinx-doc.org) to build any documentation that you create about your project.
+
+To install a compatible version of Sphinx, run the following from the project root directory:
```

Comment:
I am conflicted 🤣 since I went back and added that specifically after reading back the PR without it. The point I wanted to make was that you're using a version of Sphinx that works not a version of Sphinx that works for Kedro, which is what the `build-docs` command foists on you. Let me ruminate ...

Reply:
Got it: `kedro build-docs` installs a specific & old version of Sphinx, which means that users moving away from `kedro build-docs` might want to upgrade it. Will send a more complete rewording proposal.